cms/static/js/programs/models/api_config_model.js

```diff
@@ -0,0 +1,17 @@
+define([
+        'backbone'
+    ],
+    function( Backbone ) {
+        'use strict';
+
+        return Backbone.Model.extend({
+            defaults: {
+                username: '',
+                lmsBaseUrl: 'http://127.0.0.1:8001/',
+                programsApiUrl: 'http://127.0.0.1:8004/',
```

Comment:
I agree with @rlucioni that they should not default to local dev urls. Maybe just empty strings?